book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md

```diff
@@ -1,10 +1,10 @@
 # Duckiebot SD Card Initialization {#setup-duckiebot status=ready}
 
-This page is for the `DB18` configuration and above. 
+This page is for the `DB18` configuration and above (including Jetson Nano configurations). 
 
 <div class='requirements' markdown="1">
 
-Requires: An SD card of size at least 16 GB.
+Requires: An SD card of size at least 16 GB. 32 GB preferred. 
```

Comment:
32GB is *required* for Jetson, 16 won't work.

Reply:
will a larger sd work as well?